webapp/src/main/webapp/vue-apps/profile-settings/components/drawers/ProfileSettingFormDrawer.vue

```diff
@@ -364,7 +364,7 @@
           {{ $t('profileSettings.button.cancel') }}
         </v-btn>
         <v-btn
-          :disabled="isSaveButtonDisabled || saving || !valid"
```

Comment:
i think that we should disable the input not the save button based on canEditMultiValued information